{{cookiecutter.repo_name}}/scripts/start_{{cookiecutter.module_name}}

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
```

Comment:
If the above makes sense, then that should also change in all these headers, as well as the "year" in this line.